cli/sawtooth_cli/admin.py

```diff
@@ -22,15 +21,19 @@
 from sawtooth_cli.admin_command.utils import ensure_directory
 
 
-def do_admin(args):
+def do_admin(args, parser):
     data_dir = ensure_directory('data', '/var/lib/sawtooth')
 
+    if args.admin_cmd is None:
```

Comment:
Addendum: Argparse handles an argument of None, but doesn't provide any useful info or help. It just provides an error.